normal_operation.md

```diff
@@ -4,67 +4,42 @@ css_id: normal_operation
 
-For example, to obtain a higher degree of assurance, and for additional validation, the OEM MAY also require the supplier's update team to send a PGP / GPG signed email to the OEM's security team listing the cryptographic hashes of the new files.
+An OEM and its suppliers MAY use any transport mechanism in order to deliver these files. For example, an OEM MAY maintain a private web portal where metadata and/or images from suppliers can be uploaded. This private server MAY be managed by either the OEM or the tier-1 supplier, and SHOULD require authentication, so that only certain users are allowed to read and/or write certain files. Alternatively, the OEM and its suppliers MAY use email or courier mail.
 
-Alternatively, the OEM MAY require that updates be transmitted via DVD delivered by bonded and insured couriers. To validate the provided files, the OEM and a known contact at the supplier MAY have a video call where the cryptographic hashes of the metadata and / or images are provided by the supplier, and confirmed by the OEM.
+If the supplier signs its own images, then it delivers all of its metadata, including delegations, and associated images. Otherwise, if the OEM signs images on behalf of the supplier, then the supplier needs to update ONLY images, and the OEM is responsible for producing signed metadata. Regardless of which party produces signed metadata, the release counters associated with images SHOULD be incremented, so that attackers who compromise the Director repository are not able to rollback to obsolete images (see the opening section of the [Security page](https://github.com/uptane/deployment-considerations/blob/master/security_considerations.md) for more on this attack.)
 
-An OEM SHOULD perform this verification even if a trusted transport mechanism is used, because the OEM cannot be sure whether the transport mechanism has been compromised. If the suppliers have signed metadata, then the OEM SHOULD verify metadata and images by checking version numbers, expiration timestamps, delegations, signatures, and hashes, so that it can be sure that the metadata matches the images.
+Regardless of the transport mechanism used to deliver images, the OEM needs to somehow ensure that the images received are authentic, and have not been altered. The OEM SHOULD verify these images using some out-of-band mechanism, so that the authenticity and integrity of these images can be double-checked. For example, to obtain a higher degree of assurance, and for additional validation, the OEM MAY also require the supplier's update team to send a PGP/GPG signed email to the OEM's security team listing the cryptographic hashes of the new files.
```

Comment:
A couple of these commas seem unnecessary to me, and the "somehow" as well.

```suggestion
Regardless of the transport mechanism used to deliver images, the OEM needs to ensure that the images received are authentic and have not been altered. The OEM SHOULD verify these images using some out-of-band mechanism so that the authenticity and integrity of these images can be double-checked. For example, to obtain a higher degree of assurance, and for additional validation, the OEM MAY also require the supplier's update team to send a PGP/GPG signed email to the OEM's security team listing the cryptographic hashes of the new files.
```